docs/source/autoscaling.rst

```diff
@@ -12,21 +12,45 @@ Enabling autoscaling
 
 In order to use autoscaling, edit your ``marathon-*.yaml`` files in your soa configs and add a ``min_instances`` and a ``max_instances`` attribute and remove the ``instances`` attribute from each instance you want to autoscale. When using autoscaling, the ``min_instances`` and ``max_instances`` attributes become the minimum and maximum (inclusive) number of marathon tasks paasta will create for your job. If autoscaling information for your service is not available in Zookeeper (e.g. you've just created your service) PaaSTA will default to creating ``min_instances`` instances.
 
-Autoscaling parameters are stored in an ``autoscaling`` attribute of your instances as a dictionary. Within the ``autoscaling`` attribute, specifying a ``method`` will allow you to specify the logic that determines when to scale up and scale down your instance. If an autoscaling method isn't provided, the ``"default"`` autoscaling method will be used.
+Autoscaling parameters are stored in an ``autoscaling`` attribute of your instances as a dictionary. Within the ``autoscaling`` attribute, setting an ``ingester`` will allow you to specify a method that determines the utilization of your service. If an ingester isn't provided, the ``"mesos_cpu_ram"`` ingester will be used. Within the ``autoscaling`` attribute, setting a ``decider`` will allow you to specify the logic that determines when to autoscale your service. If a decider isn't provided, the ``"pid"`` decider will be used. Deciders and ingesters have their own optional keyword arguments that may be placed into the ``autoscaling`` dictionary as well.
```

Comment:
I'm fine with changing 'decider' and 'ingester' to something else. utilization_provider and decision_policy sound good though I'd like to hear some other opinions/more agreement